data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,596 @@
+	
+	
+outfit "Ultracapacitor Bank"
+	category "Power"
+	cost 200000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -18
+	"energy capacity" 10800
+	"ion protection" -0.03
+	"description" "These fractal capacitors are capable of storing a large amount of energy within superconducting graphene sheets. This capacitor bank chains together six individual cells, but saves space by discarding redundant components and unnecessary structural mass."
+	"description" "Like its component cells, the capacitor bank is vulnerable to ionizing weapons, but is relatively better shielded for its size by virtue of its reduced surface area to volume ratio."
+
+
+# Engines:
+
+outfit "Magnetoplasma Drive"
+	category "Engines"
+	"cost" 30000
+	thumbnail "outfit/magnetoplasma drive"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -12
+	"thrust" 9
+	"thrusting energy" 0.6
+	"thrusting heat" 0.3
+	"reverse thrust" 9
+	"reverse thrusting energy" 0.6
+	"reverse thrusting heat" 0.3
+	"turn" 270
+	"turning energy" 0.6
+	"turning heat" 0.3
+	"drag reduction" 0.15
+	"flare sprite" "effect/wanderer flare/tiny"
+		"frame rate" 5
+	"flare sound" "plasma tiny"
+	"steering flare sprite" "effect/wanderer flare/tiny"
+		"frame rate" 5
+	"steering flare sound" "plasma tiny"
+	"reverse flare sprite" "effect/wanderer flare/tiny"
+		"frame rate" 5
+	"reverse flare sound" "plasma tiny"
+	description "This compact set of magnetoplasmadynamic thrusters can propel small ships to impressive speeds over time, but is also quite energy-hungry."
+	description "Designed to be clustered, this design is capable of full six-axis control."
+
+
+outfit `Z-330 "Spark" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 660000
+	thumbnail "outfit/fusion engine small"
+	"mass" 99
+	"outfit space" -33
+	"engine capacity" -33
+	"thrust" 108
+	"thrusting energy" 9
+	"thrusting heat" 45
+	"thrusting fuel" 0.09
+	"drag reduction" 0.3
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 15
+		"scale" 0.75
+	"flare sound" "zpinch"
+	"afterburner effect" "small remass injection"
+	"magnozzle rings" 3
+	description "Balance TBD"
+
+
+outfit `Z-660 "Zap" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 1440000
+	thumbnail "outfit/fusion engine medium"
+	"mass" 198
+	"outfit space" -66
+	"engine capacity" -66
+	"thrust" 222
+	"thrusting energy" 18
+	"thrusting heat" 90
+	"thrusting fuel" 0.18
+	"drag reduction" 0.6
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 15
+	"flare sound" "zpinch"
+	"afterburner effect" "medium remass injection"
+	"magnozzle rings" 6
+	description "Balance TBD"
+
+
+outfit `Z-990 "Arc" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/fusion engine large"
+	"mass" 297
+	"outfit space" -99
+	"engine capacity" -99
+	"thrust" 360
+	"thrusting energy" 27
+	"thrusting heat" 135
+	"thrusting fuel" 0.27
+	"drag reduction" 0.9
```

Comment:
The drag reduction is of course much less of an issue here than on the Magnetoplasma Drives, especially as bundling smaller engines won't exceed the drag reduction of larger engines, but the sheer speed one can achieve gives me pause. In my trials (testing on an Ikfar A'awoj, for maximum shenanigans-capable space) it achieves slightly better speed than the Stellar-Class Afterburner in a similar amount of engine space. I find this a bit troublesome, especially as the Avgi engine-modifier outfits allow one to supply an afterburning mode that even more completely outcompetes it due to consuming a similar amount of fuel for much, much lower heat.